server/templates/event_courts_download.html

```diff
@@ -1 +1 @@
-{% for item in items %}{{ item['dates']['start'].strftime('%H%M') }} - {{ item['name'] }}.{% if (item.get('location') or [])|length > 0 %}{% for l in item.location %} {{ l.name }}{% endfor %}{% endif %}{{"\r\n"}}{% endfor %}
+{% for item in items %}{{ item['dates']['start'].strftime('%H%M') }} - {{ item['definition_short'] }}.{% if (item.get('location') or [])|length > 0 %}{% for l in item.location %} {{ l.name }}{% endfor %}{% endif %}{{"\r\n"}}{% endfor %}
```

Comment:
If the Event does not have a description we should fall back to the Event name (as name is a required field but description is not).